Engine/platform/windows/media/video/acwavi3d.cpp

```diff
@@ -129,9 +129,8 @@ int dxmedia_play_video_3d(const char* filename, IDirect3DDevice9 *device, bool u
 
     //device->Present(NULL, NULL, 0, NULL);
 
-		while (waitingForNextTick()) {
       update_polled_stuff_if_runtime();
-		}
+      platform->Delay(10);
```

Comment:
What's this "delay(10)", is this an arbitrary number of decided somehow?

Reply:
That's a weird spot. Originally I used the same timing loops as used everywhere else. But the true drawing is done by dxmedia in, I presume, another thread. So it doesn't really matter how often we check for a key press.

Reply:
So to answer your question..I just picked 10 :)